bundles/org.smarthomej.binding.notificationsforfiretv/README.md

```diff
@@ -0,0 +1,57 @@
+# Notifications For Fire TV Binding
+
+The Notifications For Fire TV Binding provides support for sending notifications to a Fire TV from rule.
+To use the binding it is necessary to install the free app `Notifications For Fire TV` by Christian Fees on the Fire TV.
+The app is available on the Amazon app store by searching for the name.
+
+## Supported Things
+
+There is only one thing: `notification` which represents a Fire TV.
+
+## Thing Configuration
+
+There are two mandatory parameters `ip` and `host`.
+
+The `ip address` must be a valid ip address to a Fire TV.
```

Comment:
```suggestion
The `ip` address must be a valid ip address to a Fire TV.
```

Is it really necessary that this is an IP address or would a hostname also be possible?

Reply:
done